components/brave_wallet_ui/stories/wallet-extension-panels.tsx

```diff
@@ -76,7 +78,9 @@ const accounts: WalletAccountType[] = [
     id: '2',
     name: 'Account 2',
     address: '0x73A29A1da97149722eB09c526E4eAd698895bDCf',
-    balance: '311780000000000000',
+    nativeBalanceRegistry: {
+      ETH: '311780000000000000'
```

Comment:
What is our preferred format? wei or eth?

Reply:
We convert `balances` from the api response which is a `hex` value to a `numeric` value in the reducer.